arkouda/categorical.py

```diff
@@ -124,6 +124,58 @@ def from_codes(cls, codes: pdarray, categories: Strings,
         return cls(None, codes=codes, categories=categories, 
                    permutation=permutation, segments=segments, uses_all_categories=uses_all_categories)
 
+    @staticmethod
+    def from_return_msg(repMsg):
+        """
+        Return a categorical instance pointing to components created by the arkouda server.
+        The user should not call this function directly.
+
+        Parameters
+        ----------
+        repMsg : str
+            ; delimited string containing the categories, codes, permutation, and segments
+            details
+
+        Returns
+        -------
+        categorical
+            A categorical representing a set of strings and pdarray components on the server
+
+        Raises
+        ------
+        RuntimeError
+            Raised if a server-side error is thrown in the process of creating
+            the categorical instance
+        """
+        # parts[0] is "categorical". Used by the generic attach method to identify the
+        # response message as a Categorical
+
+        repParts = repMsg.split(";")
```

Comment:
I am thinking that we should stick with the convention of `+` delimited creates in the response messages. This will require gluing the strings components together for the `Strings.repParts[1]`, but I think that is worth the consistency.

Reply:
Updated to use `+` delimited string in the response message and glued the components back together for the `Strings` component in `Categorical`